test/verify/check-system-info

```diff
@@ -612,7 +612,7 @@ fi
 
         # Behaviour for non-admins
         self.login_and_go('/system/hwinfo', superuser=False)
-        b.wait_visible('#hwinfo th:contains(CPU)')
+        b._wait_present('#hwinfo th:contains(CPU)')
```

Comment:
Dito with `wait_in_text()` here, or drop this completely -- the next `wait_visible()` is enough.